dizoo/atari/config/serial/asterix/asterix_mdqn_config.py

```diff
@@ -0,0 +1,64 @@
+from copy import deepcopy
+from easydict import EasyDict
+
+asterix_mdqn_config = dict(
+    exp_name='asterix_mdqn_seed0',
+    env=dict(
+        collector_env_num=8,
+        evaluator_env_num=8,
+        n_evaluator_episode=8,
+        stop_value=20000,
+        env_id='Asterix-v0',
+        #'ALE/SpaceInvaders-v5' is available. But special setting is needed after gym make.
+        frame_stack=4,
+        manager=dict(shared_memory=True, ),
```

Comment:
remove this line, default config is True

Reply:
removed